cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js

```diff
@@ -683,7 +683,17 @@ export default {
       cy.expect(Button('Download errors (CSV)').exists());
     }
   },
-
+  verifyActionsAfterErrors(errors = true) {
+    cy.do(actions.click());
+    cy.expect([
+      Button('Download matched records (CSV)').absent(),
+      Button('Start bulk edit').absent(),
+      DropdownMenu().find(HTML('Show columns')).exists(),
```

Comment:
Please, check  "Show columns" subsection is disabled.

Reply:
I already fixed it!